cmd/jag/commands/run.go

```diff
@@ -88,7 +88,7 @@ func RunCmd() *cobra.Command {
 		Args:         cobra.ExactArgs(1),
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cfg, err := directory.GetWorkspaceConfig()
+			cfg, err := directory.GetUserConfig()
```

Comment:
I think I'd prefer having a separate device config that goes in the same directory as the user config. We use the user config to identify users that have enabled analytics, so it is nice that it simple to delete just that information.

Reply:
done.